tests/src/test/scala/cats/tests/KleisliTests.scala

```diff
@@ -93,6 +93,12 @@ class KleisliTests extends CatsSuite {
     checkAll("SemigroupK[Lambda[A => Kleisli[Option, A, A]]]", SerializableTests.serializable(kleisliSemigroupK))
   }
 
+  {
+    implicit val kleisliContravariant = Kleisli.kleisliContravariant[Option, Int]
```

Comment:
I think if it can be inferred then we should probably let it be inferred in the tests. If nothing else, it provides an extra test that we don't break implicit resolution for this instance without realizing it :)